tests/libstorage/datavolume.go

```diff
@@ -43,27 +43,6 @@ import (
 	"kubevirt.io/kubevirt/tests/util"
 )
 
-func AddDataVolumeDisk(vmi *v13.VirtualMachineInstance, diskName, dataVolumeName string) *v13.VirtualMachineInstance {
```

Comment:
Not related: `v13` is really a weird name for the kubevirt import.

Reply:
Consider that it will be dropped in future PRs, since `AddDataVolume` and `AddDataVolumeTemplate` can be replaced by `libvmi` options.